features/step_definitions/user_management_steps.rb

```diff
@@ -46,7 +46,11 @@
 end
 
 Given(/^I am not an admin user$/) do
-  @current_user = Comfy::Cms::User.create!(admin: false, email: 'test@tester.com', password: 'password')
+  @current_user = Comfy::Cms::User.create!(
```

Comment:
@Tr4pSt3R we've had to add an additional step for the next part of work, so we're going to address this change to a factory while we're touching that file.